test/packages/good/elasticsearch/transform/metadata_current/transform.yml

```diff
@@ -18,3 +18,5 @@ sync:
   time:
     field: event.ingested
     delay: 1s
+settings:
+  unattended: true
```

Comment:
Add it also in the test package `good_v2`. That package is based on the package spec 2.x